tests/unit/amo/components/TestFeaturedCollectionCard.js

```diff
@@ -55,6 +55,20 @@ describe(__filename, () => {
     expect(root.find(LandingAddonsCard)).toHaveProp('className', className);
   });
 
+  it('renders a theme class name to LandingAddonsCard if the collection contains theme addons', () => {
```

Comment:
Because the logic isn't actually tied to "if the collection contains theme addons", but rather to the value of the `isTheme` property, the test should be worded that way as well.

Reply:
okay..